apps/nextra/pages/en/network/nodes/configure/consensus-observer.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Consensus Observer"
+---
+
+import { Callout, Steps } from 'nextra/components';
+
+# Consensus Observer
+
+<Callout type="warning">
+**Public Fullnodes (PFNs)**<br />
+The configuration settings described in this document are for public fullnodes (PFNs) only.
+Consensus observer has already been enabled (by default) for validators and validator fullnodes (VFNs).
+It is not recommended to change the consensus observer configurations for validators and VFNs.
+</Callout>
+
+Consensus observer is a new data dissemination technique that reduces block synchronization time, and
+transaction latencies for Aptos fullnodes (i.e., VFNs and PFNs). It has been shown to reduce end-to-end
+transaction latencies in Mainnet by: (i) ~10% at low load (e.g., 500 TPS); (ii) ~30% at medium load
+(e.g., 5k TPS); and (iii) ~50%+ at higher loads (e.g., 10k TPS+).
```

Comment:
Please use markdown for 1, 2, 3:
```
1. ...
2. ...
3. ...
```

Reply:
I'd prefer to keep it like this 😄 The markdown and bullet points make it too verbose and take up too much space (visually).

Reply:
I think I'd rather it be visually easy to see each point, and translates better rather than having it crowded.  Other languages have different grammar and different layouts, where using markdown for lists is the preferred methodology.

Reply:
Updated the sentence to remove the individual points 😄